requirements.txt

```diff
@@ -20,4 +20,5 @@ python-dateutil==2.8.2
 python-dotenv==0.20.0
 SQLAlchemy==1.4.41
 us==2.0.2
-WTForms<3.0.0
+WTForms<4.0.0
```

Comment:
Did we want to maybe pin this rather than having it be ambiguous?

Reply:
I can do that! I just copied what dependabot suggested but that's probably a better idea